trade-finance-manager-api/src/v1/rest-mappings/filters/filterActivities.test.js

```diff
@@ -8,28 +9,28 @@ const MOCK_AUTHOR = {
 
 const mockActivities = [
   {
-    type: 'COMMENT',
+    type: ACTIVITY_TYPES.COMMENT,
     timestamp: 13345665,
     text: 'test1',
     author: MOCK_AUTHOR,
     label: 'Comment added',
   },
   {
-    type: 'OTHER',
+    type: ACTIVITY_TYPES.ACTIVITY,
     timestamp: 13345665,
     text: '',
     author: MOCK_AUTHOR,
     label: 'Other',
   },
   {
-    type: 'COMMENT',
+    type: ACTIVITY_TYPES.COMMENT,
     timestamp: 13345665,
     text: 'test2',
     author: MOCK_AUTHOR,
     label: 'Comment added',
   },
   {
-    type: 'OTHER',
+    type: ACTIVITY_TYPES.ACTIVITY,
```

Comment:
Same as above

What do you think about this being `ACTIVITY_TYPES.GENERIC_ACTIVITY` instead of `ACTIVITY_TYPES.ACTIVITY`, I think it'd read easier 

Maybe could then destructure these constants as well

Reply:
I want to keep the ACTIVITY_TYPES clear so that `ACTIVITY_TYPES.ACTIVITY = 'ACTIVITY'` and similar
I think this is a lot clearer than `` ACTIVITY_TYPES.GENERIC_ACTIVITY  = 'ACTIVITY'`. 'ACTIVITY' but is legacy and would need a migration to change this enum